src/client/components/ui/Form/Select/SelectMultiple.tsx

```diff
@@ -218,68 +212,61 @@ export function SelectMultiple(props: SelectMultipleProps & { labelledBy: string
 
 	return (
-		<div>
-			<div className={styles.container}>
-				<Input
-					ref={firstInputHiddenRef}
-					onInvalid={onInvalidProps}
-					tabIndex={-1}
-					className={styles.inputHiddenValue}
-					required={required}
-					aria-hidden="true"
-					name={name}
-					value={optionsSelectedValues[0] || ''}/>
-				{optionsSelectedValues.slice(1).map((optionValue) => {
-					return <Input
+		<SelectProvider value={{
+			activeDescendant: state.activeDescendant,
+			isCurrentItemSelected,
+			onOptionSelection: selectOption,
+		}}>
+			<div>
+				<div className={styles.container}>
+					<Input
+						ref={firstInputHiddenRef}
+						onInvalid={onInvalidProps}
 						tabIndex={-1}
 						className={styles.inputHiddenValue}
-						key={optionValue}
+						required={required}
 						aria-hidden="true"
 						name={name}
-						value={optionValue}
-					/>;
-				})}
-				<div
-					role="combobox"
-					className={classNames(styles.combobox)}
-					aria-controls={listboxId}
-					aria-haspopup="listbox"
-					aria-expanded={state.isListOptionsOpen}
-					aria-labelledby={labelledBy}
-					data-touched={touched}
-					tabIndex={0}
-					onClick={() => dispatch(new SelectMultipleAction.ToggleList())}
-					aria-activedescendant={state.activeDescendant}
-					onKeyDown={onKeyDown}
-					onBlur={onBlur}
-					{...rest}
-				>
-					<PlaceholderSelectedOptions/>
-					{state.isListOptionsOpen ? <Icon name={'angle-up'}/> : <Icon name={'angle-down'}/>}
-				</div>
-				<ul
-					role="listbox"
-					ref={listboxRef}
-					aria-labelledby={labelledBy}
-					id={listboxId}
-					hidden={!state.isListOptionsOpen}>
-					{optionList.map((option, index) => {
-						const optionId = `${optionsId}-${index}`;
-						return <li
-							className={classNames(styles.optionComboboxMultiple, state.activeDescendant === optionId ? styles.optionVisuallyFocus : '')}
-							id={optionId}
-							role="option"
-							key={index}
-							onMouseDown={onMouseDown}
-							data-value={option.valeur}
-							onClick={() => selectOption(optionId)}
-							aria-selected={isCurrentItemSelected(option.valeur)}>
-							<div className={styles.option}>{option.libellé}</div>
-						</li>;
+						value={optionsSelectedValues[0] || ''}/>
+					{optionsSelectedValues.slice(1).map((optionValue) => {
+						return <Input
+							tabIndex={-1}
+							className={styles.inputHiddenValue}
+							key={optionValue}
+							aria-hidden="true"
+							name={name}
+							value={optionValue}
+						/>;
 					})}
-				</ul>
+					<div
+						role="combobox"
+						className={classNames(styles.combobox)}
+						aria-controls={listboxId}
+						aria-haspopup="listbox"
+						aria-expanded={state.isListOptionsOpen}
+						aria-labelledby={labelledBy}
+						data-touched={touched}
+						tabIndex={0}
+						onClick={() => dispatch(new SelectMultipleAction.ToggleList())}
+						aria-activedescendant={state.activeDescendant}
+						onKeyDown={onKeyDown}
+						onBlur={onBlur}
+						{...rest}
+					>
```

Comment:
```suggestion
					<div
						role="combobox"
						className={classNames(styles.combobox)}
						aria-controls={listboxId}
						aria-haspopup="listbox"
						aria-expanded={state.isListOptionsOpen}
						aria-labelledby={labelledBy}
						data-touched={touched}
						tabIndex={0}
						onClick={() => dispatch(new SelectMultipleAction.ToggleList())}
						aria-activedescendant={state.activeDescendant}
						onKeyDown={onKeyDown}
						onBlur={onBlur}
						aria-multiselectable="true"
						{...rest}
					>
```
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-multiselectable